packages/@aws-cdk/aws-cloud9/lib/environment.ts

```diff
@@ -93,6 +93,13 @@ export interface Ec2EnvironmentProps {
    * @default - CONNECT_SSH
    */
   readonly connectionType?: ConnectionType
+
+  /**
+   * The AMI alias or SSM path of the image to use
+   *
+   * @default - Cloud9 default AMI (currently amazonlinux-1-x86_64)
```

Comment:
Let's make this a required parameter. Even though CloudFormation does provide a default value, I think it is a better use experience in the long run to have to specify a value here. Eventually the default will change (like it is now) which will be unexpected for some users.